src/index.test.js

```diff
@@ -67,6 +68,21 @@ describe('hotfix cmp.init', () => {
 		expect(window.guCmpHotFix.initialised).toBe(true);
 	});
 
+	it('warn if two versions are running simultaneously', () => {
+		const spy = jest.spyOn(global.console, 'warn');
+		cmp.init({ isInUsa: false });
+		const cmpVersions = [window.guCmpHotFix.cmp.version, 'mockedVersion'];
+		// eslint-disable-next-line prefer-destructuring
+		window.guCmpHotFix.cmp.version = cmpVersions[1];
+
+		cmp.init({ isInUsa: false });
+
+		expect(spy).toHaveBeenCalledWith(
+			'Two different versions of the CMP are running:',
+			cmpVersions,
+		);
```

Comment:
_only needed if you go with the above suggestion..._
```suggestion
		expect(
			spy,
		).toHaveBeenCalledWith('Two different versions of the CMP are running:', [
			currentVersion,
			window.guCmpHotFix.cmp.version,
		]);
```